css/revops.css

```diff
@@ -29,65 +29,100 @@ p {
   line-height: 25px;
 }
 
+.hello-section {
+  min-height: 400px;
+  height: 50vh;
+  background-image: linear-gradient(6deg, #2c3e50 30%, #455d75 90%);
+}
+
 .expertise-section {
+  position: relative;
+  overflow: hidden;
   color: var(--deranged-dark-blue);
   padding: 4em;
-  padding-top: 6em;
+  padding-top: max(4em, min(9vw, 232px));
+  padding-bottom: 0;
   background-color: #ecf0f1;
-  /* background-image: linear-gradient(
-    10deg,
-    #ecf0f1 0%,
-    #ecf0f1 60%,
-    #2c3e50 60.3%,
-    #2c3e50 100%
-  ); */
+}
+.expertise-section:before,
+.expertise-section:after,
+.choose-section:before,
+.choose-section:after {
+  content: "";
+  position: absolute;
+  top: min(9vw, 232px);
+  left: 50%;
+  width: 100vw;
+  height: 100vw;
+  background-color: var(--deranged-dark-blue);
+  transform: rotate(190deg);
+  transform-origin: top left;
+}
+
+.expertise-section:after {
+  transform: rotate(260deg);
 }
 
 .expertise-container {
   margin-left: auto;
   margin-right: auto;
   max-width: 900px;
-  padding-top: 2em;
-  padding-bottom: 1em;
+  padding-top: 6em;
+  padding-bottom: 2em;
 }
 
 .services-section {
   position: relative;
   overflow: hidden;
   color: var(--deranged-text);
   background-color: var(--deranged-dark-blue);
+  background-image: linear-gradient(#2c3e50 30%, #455d75 90%);
   padding: 4em;
-  padding-top: 6em;
+  padding-top: max(6em, min(9vw, 232px));
   padding-bottom: 0;
 }
 
-.services-section:before, .services-section:after {
-  content: '';
+.services-section:before,
+.services-section:after {
+  content: "";
   position: absolute;
-  top: 0;
+  top: min(9vw, 232px);
   left: 50%;
   width: 100vw;
   height: 100vw;
   background-color: #ecf0f1;
-  transform: rotate(170deg);
+  transform: rotate(190deg);
   transform-origin: top left;
 }
 
-.services-section:after {
-  transform: rotate(280deg);
+.services-section:after,
+.choose-section:after {
+  transform: rotate(260deg);
 }
 
 .service-container-top {
   margin-left: auto;
   margin-right: auto;
   max-width: 900px;
-  padding-top: 2em;
+  padding-top: 6em;
   padding-bottom: 1em;
 }
 
+.section-button-middle {
+  display: flex;
+}
+
+a.button {
+  width: auto;
+  margin-left: auto;
+  margin-right: auto;
```

Comment:
Using auto-margins to center is generally not recommended anymore. You are inside a flex-box, so on `section-button-middle` you could just set `align-items: center` and `justify-content: center` to center everything inside the box. This centers everything horizontally *and* vertically. Remove one of the attributes to only have one of the effects.

Reply:
Agreed, fixed